ci_framework/roles/install_yamls/tasks/main.yml

```diff
@@ -119,6 +119,13 @@
   ansible.builtin.debug:
     var: cifmw_generate_makes
 
+- name: Ensure parameters directory exists
+  tags:
+    - bootstrap
+  ansible.builtin.file:
```

Comment:
care to move that at the top, in the same task creating the rest of the tree on L17?